### ISSUE TITLE
Does the "MSM8952" in devices.md mean MSM8956?

### ISSUE THREAD

**Issue by Icenowy** [msm8916-mainline/lk2nd#406]
According to my knowledge (and what https://wiki.postmarketos.org/wiki/Qualcomm_Snapdragon_650/652/653_(MSM8956/MSM8976) ) says, Xiaomi kenzo and Sony suzu/kugo are MSM8956 devices; however they're listed in devices.md as part of lk2nd-msm8952 ...

Or are these SoCs so similar that they're sharing lk2nd binary?

**Comment by TravMurav** [comment]
> Or are these SoCs so similar that they're sharing lk2nd binary?

Exactly. `lk2nd-msm8952` is a build target based on caf `msm8952` build target, which is compatible with multiple "similar enough" socs. You can consult [targets.md](https://github.com/msm8916-mainline/lk2nd/blob/main/Documentation/targets.md) to see a rough mapping of chipsets to targets. (this table is not fully accurate since i.e. we use msm8952 for msm8937 as well)

You can also confirm that by looking at the platform code which provides per-soc checks:

https://github.com/msm8916-mainline/lk2nd/blob/3368cb5894a60308d5ad088edd43bb5cbb3a5d7e/platform/msm8952/platform.c#L205-L369
